code/modules/antagonists/changeling/changeling.dm

```diff
@@ -516,6 +516,10 @@
 	user.socks = chosen_prof.socks
 
 	chosen_dna.transfer_identity(user, 1)
+	if (DIGITIGRADE in chosen_dna.species.species_traits)
+		user.Digitigrade_Leg_Swap(FALSE)
+	else
+		user.Digitigrade_Leg_Swap(TRUE)
```

Comment:
```suggestion
	user.Digitigrade_Leg_Swap(!(DIGITIGRADE in chosen_dna.species.species_traits))
```

Reply:
...what

this was what my last PR was, and you asked for the changes I made to this PR, why